contributors/guide/contributor-cheatsheet/README-ko.md

```diff
@@ -0,0 +1,398 @@
+# 쿠버네티스 컨트리뷰터 참고 자료(Cheat Sheet)
+
+쿠버네티스에 기여할 때, 일반적인 자원, 팁, 요령과
+쿠버네티스 프로젝트 내에서 사용되는 일반적인 모범 사례의 리스트이다.
+이 문서는 GitHub 컨트리뷰션 경험을 더 좋게 만들기 위한 "TL;DR" 또는
+유용한 정보의 빠른 레퍼런스이다.
+
+**목차**
+- [도움이 되는 자원](#도움이-되는-자원)
+  - [시작하기](#시작하기)
+  - [SIG 및 기타 그룹](#SIG-및-기타-그룹)
+  - [커뮤니티](#커뮤니티)
+  - [중요한 이메일 별칭](#중요한-이메일-별칭)
+  - [작업 흐름](#작업-흐름)
+  - [테스트](#테스트)
+  - [다른 유용한 링크](#다른-유용한-링크)
+- [GitHub에서 효과적으로 의사 소통하기](#GitHub에서-효과적으로-의사-소통하기)
+  - [서로에게 훌륭한 방법](#서로에게-훌륭한-방법)
+    - [좋은/나쁜 의사 소통 예시](#좋은/나쁜-의사-소통-예시)
+- [컨트리뷰션 전송](#컨트리뷰션-전송)
+  - [CLA 서명](#CLA-서명)
+  - [이슈의 오픈 및 응답](#이슈의-오픈-및-응답)
+    - [이슈 생성](#이슈-생성)
+    - [이슈 응답](#이슈-응답)
+  - [풀 리퀘스트 오픈](#풀-리퀘스트-오픈)
+    - [풀 리퀘스트 생성](#풀-리퀘스트-생성)
+    - [PR 설명 예시](#PR-설명-예시)
+    - [풀 리퀘스트 문제 해결](#풀-리퀘스트-문제-해결)
+  - [레이블](#레이블)
+- [로컬에서 작업](#로컬에서-작업)
+  - [브랜치 전략](#브랜치-전략)
+    - [Upstream 추가](#Upstream-추가)
+    - [Fork를 동기화 상태로 유지하기](#Fork를-동기화-상태로-유지하기)
+  - [스쿼시 커밋](#스쿼시-커밋)
+
+---
+
+## 도움이 되는 자원
+
+### 시작하기
+
+- [컨트리뷰터 가이드] - 어떻게 쿠버네티스 프로젝트에 기여할 수 있는지에 대한
+  가이드.
+- [개발자 가이드] - 쿠버네티스 프로젝트에 직접적으로 코드를 기여하기 위한 방법을 제공하는
+  가이드.
+- [보안과 정보 공개] - 취약점 리포트와 보안 릴리스 프로세스에 대한
+  가이드.
+
+### SIG 및 기타 그룹
+
+- [마스터 그룹 리스트][sigs]
+
+### 커뮤니티
+
+- [달력] - 쿠버네티스 커뮤니티 이벤트 모두 확인. (SIG/WG 미팅,
+  이벤트 등.)
+- [kubernetes-dev] - 쿠버네티스 개발 메일링 리스트.
+- [쿠버네티스 포럼] - 쿠버네티스 공식 포럼.
+- [Slack 채널] - 쿠버네티스 공식 Slack.
+- [StackOverflow] - 쿠버네티스 엔드-사용자의 질문에 답변할 수 있는 장소.
+- [YouTube 채널] - 쿠버네티스 커뮤니티 공식 채널.
+
+
+### 작업 흐름
+
+- [Gubernator 대시보드] - 주의가 필요한 수신/발신 풀 리퀘스트
+  확인.
+- [Prow] - 쿠버네티스 CI/CD 시스템.
+- [Tide] - 병합과 테스트를 관리하는 Prow 플러그인. [Tide 대시보드]
+- [Bot 명령] - 쿠버네티스 Bot과 상호 작용하는데 사용하는 명령. (예시:
+  `/cc`, `/lgtm`, `/retest`)
+- [GitHub 레이블] - 쿠버네티스 프로젝트 전체에서 사용되는 레이블 목록
+- [쿠버네티스 코드 검색], [@dims]가 관리.
```

Comment:
Just a very minor note since I know Dims :)

In Korean, "[A / noun], [B / explaining phrase]" is usually translated into "B A" (e.g., "B하는 A", "B에 대한 A", "B의 A"). How about this perspective? Still looks as current status, but want to discuss your opinion on it.